metrics-datadog/src/main/java/org/coursera/metrics/datadog/transport/HttpTransport.java

```diff
@@ -33,12 +37,14 @@
   private final int socketTimeout;      // in milliseconds
   private final HttpHost proxy;
   private final Executor executor;
+  private final boolean useCompression;
 
-  private HttpTransport(String apiKey, int connectTimeout, int socketTimeout, HttpHost proxy, Executor executor) {
+  private HttpTransport(String apiKey, int connectTimeout, int socketTimeout, HttpHost proxy, Executor executor, boolean useCompression) {
```

Comment:
Style nit: this already exceeded the 100 character line length. Can you break it up, please?